src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -303,19 +303,21 @@
     
     <!-- Used as an operand to bind OperandConversion -->
     <Field Name="OperandPlaceholder" Type="BoundValuePlaceholder?" SkipInVisitor="true" Null="allow"/>
-    <Field Name="OperandConversion" Type="BoundConversion?" SkipInVisitor="true" Null="allow"/>
+    <!-- Could be a BoundConversion or, in some cases of an identity conversion, the OperandPlaceholder, or null for no conversion case -->
```

Comment:
> null for no conversion case

Is this only the case for an error scenario?

Reply:
> Is this only the case for an error scenario?

Not necessary. It looks like "dynamic" uses them for non-error cases.